### ISSUE TITLE
Commit body text isn't rendered in the release notes

### ISSUE THREAD

**Issue by kevinrenskers** [loopwerk/tag-changelog#14]
[This commit](https://github.com/loopwerk/Saga/commit/71a3bd17ae1fcff20d285f72d17742c131e2ae31) has a title and a body text, but the [release notes](https://github.com/loopwerk/Saga/releases/tag/1.2.0) only show the title. What happened to the body? Why is it not rendered?

**Comment by kevinrenskers** [comment]
Ah, the default config simply doesn't render the body text:

``` javascript
renderTypeSection: function (label, commits) {
  let text = `\n## ${label}\n`;

  commits.forEach((commit) => {
    const scope = commit.scope ? `**${commit.scope}:** ` : "";
    text += `- ${scope}${commit.subject}\n`;
  });

  return text;
},
```

The fix is really simple, just add these lines: 

``` javascript
if (commit.body) {
  text += `${commit.body}\n`;
}
```

However, I am not sure if this should be part of the default config. Many commits have a body text with some kind of explanation of what was done, which is not interesting for the changelog. Or it has something like "Closes #14", which also doesn't make sense in the changelog.

So for now I am not making a change to tag-changelog's default config but I'll keep this issue open so people can chime in with their thoughts.

**Comment by kevinrenskers** [comment]
I could add a config option `includeCommitBody` which defaults to `false`, and add these lines to `renderTypeSection`:

```
if (commit.body && includeCommitBody) {
  text += `${commit.body}\n`;
}
```

So people who want to render the body text would only have to change a single boolean instead of having to overwrite a template.
